go.mod

```diff
@@ -42,9 +42,9 @@ require (
 	github.com/tendermint/btcd v0.1.1
 	github.com/tendermint/crypto v0.0.0-20191022145703-50d29ede1e15
 	github.com/tendermint/go-amino v0.16.0
-	github.com/tendermint/tendermint v0.34.21
+	github.com/tendermint/tendermint v0.37.0-alpha.1
```

Comment:
I'm guessing this would not be in v13. Does it make sense to have a separate branch for ABCI++ feature so that we don't need to deal with a lot of reverts pre-v13?

Reply:
The question is, do we want Tendermint 0.37 in Osmosis v13? If not, then yes, this will need to go to a feature branch.

cc @ValarDragon